adhocracy-plus/templates/a4modules/module_detail.html

```diff
@@ -53,7 +53,7 @@
     {% endblock %}
 
     {% block content %}
-        <div class="project-header{% if project.image %} project-header--image{% endif %} row" style="{% if view.project.image %}
+        <!-- <div class="project-header{% if project.image %} project-header--image{% endif %} row" style="{% if view.project.image %}
```

Comment:
@fuzzylogic2000 do we really not need any of this or have I forgotten something super obvious?